projects/components/src/menu-dropdown/menu-dropdown.component.scss

```diff
@@ -35,5 +36,5 @@
 }
 
 .dropdown-content {
-  @include popover-menu-content;
+  @include body-1-regular;
```

Comment:
is this the change we really want?
These are two completely different styles.
cc: @arjunlalb

Reply:
We need both here. We need the popover styling + a styling for the text.

@Greedy-Geek do not remove the existing container styling. Also, please test the changes before sending out a PR.

Reply:
Oh okay, I checked the changes before sending for the PR; maybe I missed some obvious things.